### PR TITLE
Add Docker support for Scalar DB Server

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: server/build.gradle [hunk 1]
```diff
@@ -36,6 +38,11 @@ application {
     startScripts.applicationName = 'scalardb-server'
 }
 
+docker {
+    name "ghcr.io/scalar-labs/scalardb-server:$version"
+    files tasks.distTar.outputs, 'conf/database.properties.tmpl', 'conf/log4j.properties'
```
Comment (ymorimo) [inline server/build.gradle:43]: The template file `conf/database.properties.tmpl` seems to have been removed.

File: server/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Run
+
+This runs the Scalar DB Server (you need to specify your local `database.properties` path with `-v` flag):
+
+```
+$ docker run -v <your local database.properties path>:/scalardb/server/database.properties -d -p 60051:60051 ghcr.io/scalar-labs/scalardb-server:<version>
+
+# For DEBUG logging
+$ docker run -v <your local database.properties path>:/scalardb/server/database.properties -e JAVA_OPTS=-Dlog4j.logLevel=DEBUG -d -p 60051:60051 ghcr.io/scalar-labs/scalardb-server:<version>
```
Comment (ymorimo) [inline server/README.md:38]: 👍

File: server/build.gradle (hunk 1 above)
Comment (brfrn169) [reply server/build.gradle:43]: Good catch! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Better docker file! Thank you!

**Comment by feeblefakie** [comment]
@ymorimo Do you want to take another look?

**Review by ymorimo — APPROVED** [review]
LGTM!

**Pull request merged**